### PR TITLE
Homework1 - fixes after rev.

### PR DESCRIPTION
**Pull request by marcindus**

### CODE REVIEW

File: shm/Map.cpp [hunk 1]
```diff
@@ -0,0 +1,51 @@
+
+#include <iostream>
+
+#include "Map.hpp"
+#include "ShmExceptions.hpp"
```
Comment (WBigaj) [inline shm/Map.cpp:5]: Generally order of includes are: file header, standard library include then other files from project. So it should look like tihs:
#include "Map.hpp"

#include iostream

#include "ShmExceptions.hpp"

File: shm/Ship.cpp [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+[[nodiscard]] Cargo* Ship::getCargo(size_t index) const
+{
+    try
+    {
+        return storage_.at(index);
+    }
+    catch (const std::out_of_range& ex)
+    {
+        return nullptr;
+    }
```
Comment (WBigaj) [inline shm/Ship.cpp:48]: Very smart! I haven't thought about it.

**Review by WBigaj — COMMENTED** [review]
I have checked your code and left my feedback.

File: shm/Ship.cpp (hunk 2 above)
Comment (nauka-programowania-MA) [inline shm/Ship.cpp:48]: or just validate index, to avoid exception

**Review by nauka-programowania-MA — APPROVED** [review]

**Comment by nauka-programowania-MA** [comment]
{
"requirement_ids": [2568, 2568, 2568, 2568, 2568, 2568, 2568, 2568],
"points": [5, 5, 5, 5, 5, 5, 5, 5]
}

**Pull request closed without merging**